### PR TITLE
JAVA-2662: Document how to enable JMX connection pool monitoring

### PR DESCRIPTION
**Pull request by jyemin**
https://jira.mongodb.org/browse/JAVA-2662

### CODE REVIEW

File: docs/reference/content/driver-async/reference/monitoring.md [hunk 1]
```diff
@@ -33,6 +33,15 @@ application has multiple `MongoClient` instances connected to the same MongoDB s
 - `waitQueueSize`: the current size of the wait queue for a connection from this pool
 - `checkedOutCount`: the current count of connections that are currently in use
 
+JMX connection pool monitoring is disabled by default. To enable it, an application adds a
+`com.mongodb.management.JMXConnectionPoolListener` instance as a `ConnectionPoolListener` via `MongoClientSettings`:
```
Comment (rozza) [inline docs/reference/content/driver-async/reference/monitoring.md]: > To enable it, an application adds a `com.mongodb.management.JMXConnectionPoolListener` instance as a `ConnectionPoolListener` via `MongoClientSettings`:

Seems overly complex, suggest:

> To enable it add a `com.mongodb.management.JMXConnectionPoolListener` instance via `MongoClientSettings`:

File: docs/reference/content/driver/reference/monitoring.md [hunk 2]
```diff
@@ -33,6 +33,18 @@ application has multiple `MongoClient` instances connected to the same MongoDB s
 - `waitQueueSize`: the current size of the wait queue for a connection from this pool
 - `checkedOutCount`: the current count of connections that are currently in use
 
+JMX connection pool monitoring is disabled by default. To enable it, an application adds a
```
Comment (rozza) [inline docs/reference/content/driver/reference/monitoring.md]: As above.

File: docs/reference/content/driver-async/reference/monitoring.md (hunk 1 above)
Comment (jyemin) [reply docs/reference/content/driver-async/reference/monitoring.md]: Done

File: docs/reference/content/driver/reference/monitoring.md (hunk 2 above)
Comment (jyemin) [reply docs/reference/content/driver/reference/monitoring.md]: Done

**Pull request closed without merging**